<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="testcases.LoginTest" duration-ms="58093" started-at="2020-03-14T04:45:18Z" finished-at="2020-03-14T04:46:16Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="58093" started-at="2020-03-14T04:45:18Z" finished-at="2020-03-14T04:46:16Z">
      <class name="testcases.LoginTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:testcases.LoginTest@13fee20c]" name="setUp" is-config="true" duration-ms="29293" started-at="2020-03-14T05:45:18Z" finished-at="2020-03-14T05:45:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginPageTitleTest()[pri:1, instance:testcases.LoginTest@13fee20c]" name="loginPageTitleTest" duration-ms="9" started-at="2020-03-14T05:45:47Z" finished-at="2020-03-14T05:45:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [S�identifier sur LinkedIn | LinkedIn] but found [S’identifier sur LinkedIn | LinkedIn]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [S�identifier sur LinkedIn | LinkedIn] but found [S’identifier sur LinkedIn | LinkedIn]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at testcases.LoginTest.loginPageTitleTest(LoginTest.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageTitleTest -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:testcases.LoginTest@13fee20c]" name="TearDown" is-config="true" duration-ms="729" started-at="2020-03-14T05:45:48Z" finished-at="2020-03-14T05:45:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:testcases.LoginTest@13fee20c]" name="setUp" is-config="true" duration-ms="14034" started-at="2020-03-14T05:45:48Z" finished-at="2020-03-14T05:46:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginTest()[pri:2, instance:testcases.LoginTest@13fee20c]" name="loginTest" duration-ms="12530" started-at="2020-03-14T05:46:02Z" finished-at="2020-03-14T05:46:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:testcases.LoginTest@13fee20c]" name="TearDown" is-config="true" duration-ms="1298" started-at="2020-03-14T05:46:15Z" finished-at="2020-03-14T05:46:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- testcases.LoginTest -->
    </test> <!-- Command line test -->
  </suite> <!-- testcases.LoginTest -->
</testng-results>
